app/utils/report/templates/test.txt

```diff
@@ -11,7 +11,7 @@ Test results summary
   Commit:  {{ git_commit }}
 
 {% for t in test_groups|sort(attribute='name') %}
-{{ "%-2s | %-22s | %-5s | %3s total: %3s PASS %3s FAIL %3s SKIP"|format(loop.index, t.board, t.arch, t.total_tests, t.total["PASS"], t.total["FAIL"], t.total["SKIP"]) }}
+{{ "%-2s | %-22s | %-10s | %-5s | %3s total: %3s PASS %3s FAIL %3s SKIP"|format(loop.index, t.board, t.build_environment, t.arch, t.total_tests, t.total["PASS"], t.total["FAIL"], t.total["SKIP"]) }}
```

Comment:
`%-8s` should be enough to show things like `clang-xx`, the table is already quite wide.

Reply:
corrected